src/main/java/com/google/codeu/data/Post.java

```diff
@@ -44,95 +44,46 @@ public UUID getPostID() {
 		return postID;
 	}
 
-	/**
-	 * @param postID the postID to set
-	 */
-	public void setPostID(UUID postID) {
-		this.postID = postID;
-	}
-
 	/**
 	 * @return the creator
 	 */
 	public String getCreator() {
 		return creator;
 	}
 
-	/**
-	 * @param creator the creator to set
-	 */
-	public void setCreator(String creator) {
-		this.creator = creator;
-	}
-
 	/**
 	 * @return the timestamp
 	 */
 	public long getTimestamp() {
 		return timestamp;
 	}
 
-	/**
-	 * @param timestamp the timestamp to set
-	 */
-	public void setTimestamp(long timestamp) {
-		this.timestamp = timestamp;
-	}
-
 	/**
 	 * @return the coverInamgeUrl
 	 */
 	public String getCoverInamgeUrl() {
 		return coverInamgeUrl;
 	}
 
-	/**
-	 * @param coverInamgeUrl the coverInamgeUrl to set
-	 */
-	public void setCoverInamgeUrl(String coverInamgeUrl) {
-		this.coverInamgeUrl = coverInamgeUrl;
-	}
-
 	/**
 	 * @return the marker
 	 */
 	public LocationMarker getMarker() {
 		return marker;
 	}
 
-	/**
-	 * @param marker the marker to set
-	 */
-	public void setMarker(LocationMarker marker) {
-		this.marker = marker;
-	}
-
 	/**
 	 * @return the chartData
 	 */
 	public List<ChartDataRow> getChartData() {
 		return chartData;
 	}
 
-	/**
-	 * @param chartData the chartData to set
-	 */
-	public void setChartData(List<ChartDataRow> chartData) {
-		this.chartData = chartData;
-	}
-
 	/**
 	 * @return the mesages
 	 */
 	public List<Message> getMesages() {
 		return mesages;
 	}
 
-	/**
-	 * @param mesages the mesages to set
-	 */
-	public void setMesages(List<Message> mesages) {

```

Comment:
ahh so we need this one still. since we are setting the messages after the the fact of post creation. or maybe instead of this we can do something like `addMessages` since we are only ever adding messages one at at time.